tests-fuzz/src/ir.rs

```diff
@@ -496,6 +536,31 @@ pub fn replace_default(
     new_rows
 }
 
+/// Sorts a vector of rows based on the values in the specified primary key columns.
+pub fn sort_by_primary_keys(rows: &mut [RowValues], primary_keys_idx: Vec<usize>) {
+    rows.sort_by(|a, b| {
+        let a_keys: Vec<_> = primary_keys_idx.iter().map(|&i| &a[i]).collect();
+        let b_keys: Vec<_> = primary_keys_idx.iter().map(|&i| &b[i]).collect();
+        for (a_key, b_key) in a_keys.iter().zip(b_keys.iter()) {
+            match a_key.cmp(b_key) {
+                Some(std::cmp::Ordering::Equal) => continue,
+                non_eq => return non_eq.unwrap(),
+            }
+        }
+        std::cmp::Ordering::Equal
+    });
+}
```

Comment:
There're similar code snippets in insert fuzz tests. Are they serving the same purpose? Maybe I could refactor the insert fuzz tests with this later.

Reply:
Yes, it's fine for now. we can refactor it in a separate PR later.☺️